src/main/java/com/fasterxml/uuid/impl/DbLocalityTimeBasedGenerator.java

```diff
@@ -0,0 +1,118 @@
+package com.fasterxml.uuid.impl;
+
+import java.util.UUID;
+
+import com.fasterxml.uuid.*;
+
+/**
+ * Implementation of UUID generator that uses time/location based generation
+ * method field compatible with UUIDv1, reorderd for improved DB locality
+ * (variant 6).
+ * <p>
+ * As all JUG provided implementations, this generator is fully thread-safe.
+ * Additionally it can also be made externally synchronized with other instances
+ * (even ones running on other JVMs); to do this, use
+ * {@link com.fasterxml.uuid.ext.FileBasedTimestampSynchronizer} (or
+ * equivalent).
+ *
+ * @since 3.1
+ */
+public class DbLocalityTimeBasedGenerator extends NoArgGenerator
```

Comment:
naming?

Reply:
Should then follow naming of the method, so one of `TimeBasedReversedGenerator`, ReversedTime-`, `SortedTime-`, `TimeSorted-`.